docker-compose.yml

```diff
@@ -93,6 +94,31 @@ services:
     networks:
       - reefscan
 
+  backtracking:
+    image: reef-explorer-backtracking-dev
```

Comment:
We want to reuse the image from the crawler for this service.  Therefore, we should change `reef-explorer-backtracking-dev` to `reef-explorer-crawler-dev`